docs/yul.rst

```diff
@@ -760,8 +760,14 @@ but not including position ``b``, ``storage[p]`` signifies the storage contents
 similarly, ``transientStorage[p]`` signifies the transient storage contents at slot ``p``.
 
 Since Yul manages local variables and control-flow,
-opcodes that interfere with these features are not available. This includes
-the ``dup`` and ``swap`` instructions as well as ``jump`` instructions, labels and the ``push`` instructions.
+the following opcodes that interfere with these features are not available:
+
+* all ``dup`` variants
+* all ``swap`` variants
+* all ``push`` variants
+* the ``jump`` instructions
+* ``jumpdest`` labels
+* the ``pc`` instruction
 
 +-------------------------+-----+---+-----------------------------------------------------------------+
 | Instruction             |     |   | Explanation                                                     |
```

Comment:
It's stil llisted in the instruction list, though :-).

Not sure I'd actually mention ``pc`` together with dup/swap/jump as "interfering with variables and control-flow".

Maybe just removing it from the instruction list here is enough.
We could also add a note that it was removed in 0.7.0  - and could even explain that it doesn't have well-defined semantics, since Yul doesn't have any guarantee about code layout.

But yeah, minimal version of this would just remove it from the instruction list.

Reply:
Thanks for catching that. Where would be best to add the note about it's removal?

Reply:
Either pretty much where you had it initially as a new sentence after the stack opcodes - or among the notes after the instruction table.
But yeah, I'm fine with just removing it, it's unlikely that it'll be missed and not sure how much value another note about it would have. I'll merge as is.